compiler/rustc_error_messages/locales/en-US/parser.ftl

```diff
@@ -158,3 +158,6 @@ parser_remove_let = expected pattern, found `let`
 
 parser_use_eq_instead = unexpected `==`
     .suggestion = try using `=` instead
+
+parser_use_empty_block_not_semi = expected { "`{}`" }, found `;`
+    .suggestion = try using { "`{}`" } instead
```

Comment:
Not sure if "try using `{}` instead of `;`" is better or worse?

Reply:
found it kinda redundant given the primary message of the error, but i don't care either way

Reply:
Ok sure.